packages/cli/src/storyblok/storyblok-client.ts

```diff
@@ -1,11 +1,12 @@
 import { red } from 'kleur/colors'
 import type { Response } from 'node-fetch'
 
-const PARTNER_FIELD_TYPES_API_ENDPOINT =
-  'https://mapi.storyblok.com/v1/partner_field_types/'
-
 export type FieldType = { id: number; name: string; body: string }
 
+export type Scope = 'my-space' | 'partner-portal'
```

Comment:
It think this should be `'my-plugins'`,because the plugins under "My Plugins" are personal plugins that are tied to the user account.